<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WheelDiaSet</class>
 <widget class="QWidget" name="WheelDiaSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 20px, &quot;SimHei&quot;;	
}

QWidget
{
	background-color: black;
}</string>
  </property>
  <widget class="QLabel" name="label_carNo4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>72</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_wheelDia">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>130</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>轮径设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_carNo">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>72</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>车号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>350</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;}
QPushButton:disabled
{
		border-radius:5px;
	font: 20px, &quot;SimHei&quot;;
    color: grey;
	background-color: rgb(221,221,221);
}</string>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>350</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;}
QPushButton:disabled
{
		border-radius:5px;
	font: 20px, &quot;SimHei&quot;;
    color: grey;
	background-color: rgb(221,221,221);
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_carNo2">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>72</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_carNo1">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>72</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_carNo3">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>72</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wheelDia1">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>130</y>
     <width>188</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
color: black;
background-color: rgb(250, 250, 250);
border: 2px solid rgb(117,117,117);


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wheelDia2">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>130</y>
     <width>210</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
color: black;
background-color: rgb(250, 250, 250);
border: 2px solid rgb(117,117,117);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wheelDia3">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>130</y>
     <width>210</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
color: black;
background-color: rgb(250, 250, 250);
border: 2px solid rgb(117,117,117);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wheelDia4">
   <property name="geometry">
    <rect>
     <x>801</x>
     <y>130</y>
     <width>180</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
color: black;
background-color: rgb(250, 250, 250);
border: 2px solid rgb(117,117,117);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_confirm">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>350</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;}
QPushButton:disabled
{
		border-radius:5px;
	font: 20px, &quot;SimHei&quot;;
    color: grey;
	background-color: rgb(221,221,221);
}</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_errorInfo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0;
color:red;</string>
   </property>
   <property name="text">
    <string>设置失败，轮径值非法，新轮径值必须在［770mm，840mm］区间內！</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_original3">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>101</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_original1">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>101</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_original">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>101</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>原轮径值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_original2">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>101</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_original4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>101</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
